test/gtest/ucs/test_memtype_cache.cc

```diff
@@ -198,6 +198,86 @@ class test_memtype_cache : public ucs::test_with_param<ucs_memory_type_t> {
         }
     }
 
+    struct region_info {
+        void              *start;
+        void              *end;
+        ucs_memory_type_t mem_type;
+    };
+
+    region_info make_region_info(size_t start, size_t end,
```

Comment:
could be static function, or constructor of region_info

Reply:
yes, good point
make it as ctor